src/libstd/sys/windows/process.rs

```diff
@@ -90,6 +90,12 @@ impl Command {
     pub fn cwd(&mut self, dir: &OsStr) {
         self.cwd = Some(dir.to_os_string())
     }
+    pub fn get_program(&self) -> &OsString {
+        &self.program
+    }
+    pub fn args_iter(&self) -> slice::Iter<OsString> {
```

Comment:
I think the unix version would reqiure allocation of the slice, since the underlying storage isn't `OsStr`. This also conflicts with the naming of `arg()` for the setter.